### DISCUSSION TITLE
Customize specific features

### DISCUSSION THREAD

**Discussion by amit510** [0x6d69636b/windows_hardening#23] [Q&A] [answered]
I was wondering if its possible to comment out specific things that I do not need to be changed? For example If I still want to keep the RDP enabled when I run this script in HailMary Mode and apply rest of the changes. Please let me know if there is an easy way to accomplish this?

**Comment by 0x6d69636b** [comment, marked as answer]
I recommend that you compile your own list based on one of the existing lists and set the values according to your needs. Another advantage is that you can use this list later to check whether your system is still compliant.

**Reply by amit510** [reply]
If I choose one of the list from [here](https://github.com/0x6d69636b/windows_hardening/tree/master/lists), where do I specify my own custom list? Sorry if its a too basic question, not too familiar with this script.

**Reply by 0x6d69636b** [reply]
Don't worry, just ask what you want to know. You can take a list, copy it and then you call HardeningKitty with the parameter `-FileFindingList` and point to your list. For example `Invoke-HardeningKitty -FileFindingList C:\tmp\new_list.csv`. 

If you use HailMary mode, then turn on the log so that you can better evaluate any errors. The command would then be like: `Invoke-HardeningKitty -Mode HailMary -FileFindingList C:\tmp\new_list.csv -Log` 

In the header of the script [Invoke-HardeningKitty.ps1](https://github.com/0x6d69636b/windows_hardening/blob/master/Invoke-HardeningKitty.ps1) all parameters are documented, maybe this helps. Please give me feedback if something is not clearly described, then I can improve it.

**Reply by amit510** [reply]
Thank you so much, this is exactly the info I was looking for. Will give it a try tomorrow, will report back if any issues. Keep up the good work with this awesome script.
